src/packages/@ncigdc/components/Explore/SummaryPage/HistogramCard.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import BarChart from '@ncigdc/components/Charts/BarChart';
+
+const chartStyles = {
+  bars: { fill: 'rgb(23, 132, 172)' },
+  tooltips: {
+    fill: '#fff',
+    stroke: 'rgb(144, 144, 144)',
+    textFill: 'rgb(144,144,144)',
+  },
+  xAxis: {
+    stroke: 'rgb(200, 200, 200)',
+    textFill: 'rgb(107, 98, 98)',
+  },
+  yAxis: {
+    stroke: 'rgb(200, 200, 200)',
+    textFill: 'rgb(144,144,144)',
+  },
+};
+const CHART_HEIGHT = 320;
+
+const HistogramCard = ({
+  bottomMarginForxAxisTitle,
+  data,
+  mappingLabel,
+  mappingValue,
+  xAxisTitle,
+}) => {
+  const CHART_MARGINS = {
```

Comment:
this can be extracted from the function (i.e. before line 22), so a new `const` is not created at every render.